tests/consensus_tests/test_collection_sharding.py

```diff
@@ -0,0 +1,118 @@
+import os
+import pathlib
+import shutil
+import time
+
+import requests
+
+from .utils import *
+from . import conftest
+from subprocess import Popen
+
+N_PEERS = 5
+N_SHARDS = 6
+
+def test_collection_after_peers_added(tmp_path: pathlib.Path):
```

Comment:
```suggestion
def test_collection_sharding(tmp_path: pathlib.Path):
```

Reply:
The joy of copy pasting! 
Thanks for catching this one :+1:
https://github.com/qdrant/qdrant/pull/599/commits/1637a319951836937b4818486ce7d5f1627231db